### ISSUE TITLE
Link to website from stats pages is broken

### ISSUE THREAD

**Issue by JesseWeinstein** [openinframap/openinframap#128]
Power plant stats pages like https://openinframap.org/stats/area/United%20States/plants/543344753 have an invalid (relative) link where they should have an absolute link to the external website stored in the `website` key.

**Comment by russss** [comment]
This is arguably an issue with the tagging - the [wiki page](https://wiki.openstreetmap.org/wiki/Key:website) says "Include the scheme (http or https) explicitly". Should probably still handle this better, though.

**Comment by JesseWeinstein** [comment]
Yeah, it looks like openstreetmap.org also handles such values badly -- but they are lots of them.  [This SOPHOX query](https://sophox.org/#SELECT%20%2a%20WHERE%20%7B%0A%20%20%3Fosm%20%28osmt%3Awebsite%7Cosmt%3Acontact%3Awebsite%7Cosmt%3Asource%3Awebsite%7Cosmt%3Aoperator%3Awebsite%7Cosmt%3Asource%3Aurl%7Cosmt%3Aurl%29%20%3Fwebsite.%0A%20%20FILTER%28%21STRSTARTS%28%3Fwebsite%2C%20%22http%22%29%29%0A%20%20FILTER%28%3Fwebsite%20%21%3D%20%22%22%29%0A%7D) shows over 41,000 (out of over 1.1 million, so only around 4% -- but that is still a lot!).

**Comment by russss** [comment]
Feels like something editors could do better! I'll have a think about the best way of solving this on openinframap.

**Comment by JesseWeinstein** [comment]
Probably the right way to solve it is just to insert a "http://" prefix in front of any URLs that don't have one already. Looking at the code, I'm not immediately sure what the right way to do that is, though.
